README.md

```diff
@@ -0,0 +1,74 @@
+<h1 align="center" style="margin:1em;">
+  <a href="https://iris-grib.readthedocs.io/en/latest/">iris-grib</a>
+</h1>
+
+<h4 align="center">
+GRIB interface for <a href="https://github.com/SciTools/iris">Iris</a>.
+</h4>
+
+<p align="center">
+<!-- https://shields.io/ is a good source of these -->
+<a href="https://anaconda.org/conda-forge/iris-grib">
+<img src="https://img.shields.io/conda/dn/conda-forge/iris-grib.svg"
+ alt="conda-forge downloads" /></a>
+<a href="https://github.com/SciTools/iris-grib/releases">
+<img src="https://img.shields.io/github/tag/SciTools/iris-grib.svg"
+ alt="Latest version" /></a>
+<a href="https://github.com/SciTools/iris-grib/commits/master">
+<img src="https://img.shields.io/github/commits-since/SciTools/iris-grib/latest.svg"
+ alt="Commits since last release" /></a>
+<a href="https://github.com/SciTools/iris-grib/graphs/contributors">
+<img src="https://img.shields.io/github/contributors/SciTools/iris-grib.svg"
+ alt="# contributors" /></a>
+<a href="https://travis-ci.org/SciTools/iris-grib/branches">
+<img src="https://api.travis-ci.org/repositories/SciTools/iris-grib.svg?branch=master"
+ alt="Travis-CI" /></a>
+<!-- <a href="https://zenodo.org/badge/latestdoi/5282596">
+<img src="https://zenodo.org/badge/5282596.svg"
+ alt="zenodo" /></a> -->
+</p>
+<br>
+
+# Table of contents
```

Comment:
Likewise, a webpage should **never** have more than one h1 tag. Please make this an h2 tag.